assets/rpvoicechat/lang/fr.json

```diff
@@ -0,0 +1,62 @@
+{
+    "item-handbell": "Clochette",
+    "item-royalhorn": "Cor royal",
+    "item-royalhornhandle": "Manche de cor royal",
+    "item-royalhornhead": "Bec de cor royal",
+    "item-handheldradio": "Radio portative",
+    "item-telegraphwire": "Câble de télégraphe",
+    "item-radiocase": "Boitier radio",
+
+    "itemdesc-handbell": "Une petite clochette. L'argent de la cloche semble émettre un étrange scintillement violet ...",
+
+    "block-callbell": "Sonnette",
+
+    "blockdesc-callbell": "Une sonnette utilisée pour appeler une personne.",
+
+    "Command.Shout.Desc": "Défini la distance de cri en blocs",
+    "Command.Shout.Success": "Distance de cri défini à {0}",
+    "Command.Talk.Desc": "Défini la distance de parole en blocs",
+    "Command.Talk.Success": "Distance de parole défini à {0}",
+    "Command.Whisper.Desc": "Défini la distance de chuchotage en blocs",
+    "Command.Whisper.Success": "Distance de chuchotage défini à {0}",
+    "Command.Info.Desc": "Affiche les distances audio actuelles",
+    "Command.Info.Success": "Distance de chuchotage: {0} blocs\nDistance de parole: {1} blocs\nDistance de cri: {2} blocs",
+    "Command.Reset.Desc": "Réinitialise les distances audio",
+    "Command.Reset.Success": "Distances audio réinitialisées",
+    "Hotkey.ModMenu": "RPVoice: Menu du mod",
+    "Hotkey.VoiceLevel": "RPVoice: Changer la distance de la voix voice distance",
```

Comment:
Shouldn't it be `RPVoice: Changer la distance de la voix`?

Reply:
yeah I totaly forgot to change this